src/transport/raw/MessageHeader.h

```diff
@@ -74,6 +73,18 @@ static constexpr size_t kMaxAppMessageLen = detail::kMaxApplicationPayloadAndMIC
 
 static constexpr uint16_t kMsgUnicastSessionIdUnsecured = 0x0000;
 
+#if INET_CONFIG_ENABLE_TCP_ENDPOINT
```

Comment:
Why does this need to be conditioned on INET_CONFIG_ENABLE_TCP_ENDPOINT?  These are just constexpr constants; they cost nothing....

I guess the packetbuffer constants are conditioned on it, but why is that?